helmfile/charts/fluentd-configmap/values.yaml

```
@@ -0,0 +1,28 @@
+aggregator:
+  enabled: true
+    # name of the aggregator configmap
+  name: fluentd-aggregator-configmap
+  replicaCount: 1
+  configMaps:
+    # fluentd.conf: |
+    # <source>
+      # @type prometheus
+      # port 24231
+    # </source>
+  plugins:
+    enabled: true
+    pluginsList:
+      - fluent-plugin-s3
+forwarder:
+  enabled: true
+  # name of the forwarder configmap
+  name: fluentd-forwarder-configmap
+  configMaps:
```

Missed to add this one
```suggestion
  configMaps: {}
```